### PR TITLE
Various fixes and minor enhancements.

### PR DESCRIPTION
**Pull request by setrofim**

### CODE REVIEW

File: devlib/target.py [hunk 1]
```diff
@@ -1652,6 +1652,9 @@ def __init__(self,
                                                shell_prompt=shell_prompt,
                                                conn_cls=conn_cls)
 
+    def wait_boot_complete(self, timeout=10):
+        pass
```
Comment (marcbonnici) [inline devlib/target.py]: Should this be added to `LinuxTarget`?

Comment (setrofim) [reply devlib/target.py]: Yup, added.

Comment (marcbonnici) [reply devlib/target.py]: Does this need to be in LocalLinuxTarget as well?

**Comment by marcbonnici** [comment]
Typo in commit message "acmecape: fix warning fromatting"

File: devlib/target.py (hunk 1 above)
Comment (setrofim) [reply devlib/target.py]: Good point. Removed from `LocalLinuxTarget`

**Comment by setrofim** [comment]
Typo fixed.

**Pull request merged**